Cargo.toml

```diff
@@ -1,30 +1,32 @@
 [package]
-name    = "ulid"
+name    = "pgx_ulid"
 publish = false
-version = "0.1.5"
+version = "0.2.0"
 
 edition      = "2021"
-rust-version = "1.74.0"
+rust-version = "1.81.0"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "lib"]
+
+[[bin]]
+name = "pgrx_embed_pgx_ulid"
+path = "./src/bin/pgrx_embed.rs"
 
 [features]
 default = ["pg15"]
-pg11    = ["pgrx-tests/pg11", "pgrx/pg11"]
-pg12    = ["pgrx-tests/pg12", "pgrx/pg12"]
-pg13    = ["pgrx-tests/pg13", "pgrx/pg13"]
 pg14    = ["pgrx-tests/pg14", "pgrx/pg14"]
 pg15    = ["pgrx-tests/pg15", "pgrx/pg15"]
 pg16    = ["pgrx-tests/pg16", "pgrx/pg16"]
+pg17    = ["pgrx-tests/pg17", "pgrx/pg17"]
 pg_test = []
 
 [dependencies]
-inner_ulid = { package = "ulid", version = "1.0.0" }
-pgrx       = "^0.11.2"
+inner_ulid = { package = "ulid", version = "1.1.3" }
+pgrx       = "0.12.7"
```

Comment:
I think the `^` is needed, otherwise, I wouldn't have left it here. IIRC, there was an issue and therefore it was added by a contributor.